app/src/component-library/helpers/classNames/index.js

```diff
@@ -0,0 +1,22 @@
+export const ClassNames = ({ blk = '', elt = '', mods = [''] }) => {
+
+    let subClass = '';
+
+    if (blk !== '') {
+        if (elt !== '') {
+            subClass = ` ${blk} ${blk}__${elt}`
+        }
+
+        if (mods.length > 0) {
+            mods.map((item) => {
+                if (item !== '')
+                    subClass += ` ${blk}__${elt}--${item}`
+            })
+        }
+        return subClass;
+    }
```

Comment:
what happens if the input is `{ blk: 'abc' }`

Reply:
The output is: abc